### PR TITLE
Created Lichen for ScienceRun1. Updated AmBe FV. Created NG FV

### PR DESCRIPTION
**Pull request by jpienaar13**
Corrected definition of AmBe FV in sciencerun0 to reflect the correct source position, as mentioned by Dominic: https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:analysis:dominick:sr1_ambe_check

Created NG FV. (Not sure if needed, but at least it's there if we do):
https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:analysis:dominick:ng_early_look

### CODE REVIEW

File: lax/lichens/sciencerun1.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    string = "(distance_to_source < 103.5) & (-92.9 < z) & (z < -9) & (sqrt(x*x + y*y) < 42.00)"
+
+    def pre(self, df):
+        source_position = (97, 43.5, -50)
+        df.loc[:, 'distance_to_source'] = ((source_position[0] - df['x']) ** 2 +
+                                           (source_position[1] - df['y']) ** 2 +
+                                           (source_position[2] - df['z']) ** 2) ** 0.5
+        return df
+    
+class AmBeFiducial(StringLichen):
```
Comment (pdeperio) [inline lax/lichens/sciencerun1.py]: Repeated definition?  (also shows up in codacy check below, please go through this)

**Review by pdeperio — CHANGES_REQUESTED** [review]

**Comment by jpienaar13** [comment]
Removed the doubled definition

**Review by pdeperio — DISMISSED** [review]

**Comment by pdeperio** [comment]
Should the ```sciencerun0.py``` also be updated with the new AmBe FV?

Also, please click "Details" next to failed "codacy/pr" check, and fix as many as you can.

**Comment by jpienaar13** [comment]
Will do. We can update sciencerun0. I see no reason why that would not be a good idea. @andrewgrahambrown any objections

File: lax/lichens/sciencerun1.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+        # Replaces Fiducial
+        self.lichen_list[0] = NGFiducial()
+
+        
+DAQVeto = sciencerun0.DAQVeto
+
+S2Tails = sciencerun0.S2Tails
+
+FiducialCylinder1T = sciencerun0.S2Tails
```
Comment (pdeperio) [inline lax/lichens/sciencerun1.py]: Typo?

File: lax/lichens/sciencerun1.py [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+        self.lichen_list[1] = S1LowEnergyRange()
+
+        # Use a simpler single scatter cut
+        self.lichen_list[5] = S2SingleScatterSimple()
+
+        self.lichen_list += [
+            S1PatternLikelihood(),
+            S2Width(),
+            S1MaxPMT(),
+            SingleElectronS2s()
```
Comment (pdeperio) [inline lax/lichens/sciencerun1.py]: @feigaodm: is this still needed after https://github.com/XENON1T/pax/pull/576?

If not, I guess we should remove from SR0 lichen too once SR0 gets reprocessed?

**Review by pdeperio — CHANGES_REQUESTED** [review]

**Comment by pdeperio** [comment]
@jpienaar13 please also commit the AmBe FV SR0 fix to this branch.

File: lax/lichens/sciencerun1.py (hunk 2 above)
Comment (jpienaar13) [reply lax/lichens/sciencerun1.py]: yes, that looks like a typo

**Comment by jpienaar13** [comment]
@pdeperio 

Do we want to keep the old incorrect volume, or just update SR0 AmBe fiducial to the correct one?

**Comment by pdeperio** [comment]
Just update with new version number.

**Comment by jpienaar13** [comment]
Done

**Comment by tunnell** [comment]
@jpienaar13 I think you accidentally introduced style error

**Comment by jpienaar13** [comment]
@tunnell That comment is a little cryptic.

**Comment by tunnell** [comment]
@jpienaar13 do you see how it says that a check had failed?  Also, @pdeperio needs to say if you satisfied the changes requested in the review.  

@jpienaar13 But yeah, you see where it says "All checks have failed", then "codacy/pr", on the right is "details" that takes you here:

https://www.codacy.com/app/weiyuehuan/lax/pullRequest?prid=694964

It's just trailing white space on line 94 so tiny to fix.

**Review by tunnell — APPROVED** [review]

**Comment by jpienaar13** [comment]
@tunnell Eh,  I thought I fixed that one yesterday.

**Comment by tunnell** [comment]
You can just edit the file on the github website.

**Comment by tunnell** [comment]
https://github.com/XENON1T/lax/edit/ScienceRun1_lichens/lax/lichens/sciencerun1.py?pr=/XENON1T/lax/pull/58

**Review by pdeperio — APPROVED** [review]

**Comment by pdeperio** [comment]
@tunnell seems like the codacy report doesn't show all the issues at once (they appear one at a time as you fix online).  so i just did ```make lint``` locally and fixed a bunch manually.

**Comment by pdeperio** [comment]
@jpienaar13: @tunnell noticed that maybe this corrected AmBe FV should have gone into ```sciencerun0``` instead, then inherited here. Is this correct?

**Comment by jpienaar13** [comment]
That is correct yes.

**Comment by pdeperio** [comment]
ah he opened an issue here https://github.com/XENON1T/lax/issues/95

**Pull request merged**